utils_cv/action_recognition/model.py

```diff
@@ -233,37 +175,36 @@ def fit(self, train_cfgs):
             )
 
         # Learning rate scheduler
-        if train_cfgs.get('use_one_cycle_policy', False):
+        if train_cfgs.get("use_one_cycle_policy", False):
```

Comment:
Fyi, the parameters I found to work well for HMDB-51 (and now already used in the master 01 notebook) were for simple StepLR because our older pytorch version does not support OneCycleLR. Should we update pytorch we could re-visit and see if OneCycleLR performs better (but would have to tune at least the learning rate then again).